test/unit/org/apache/cassandra/db/ReadResponseTest.java

```diff
@@ -162,12 +162,8 @@ public void makeDigestDoesntConsiderRepairedDataInfo()
     }
 
     private void verifySerDe(ReadResponse response) {
-        // check that roundtripping through ReadResponse.serializer behaves as expected.
-        // ReadResponses from pre-4.0 nodes will never contain repaired data digest
-        // or pending session info, but we run all messages through both pre/post 4.0
-        // serde to check that the defaults are correctly applied
+        // check that roundtripping through ReadResponse.serializer behaves as expected
         roundTripSerialization(response, MessagingService.current_version);
```

Comment:
`roundTripSerialization` now always takes the same version argument. I'd either inline it or annotate the method with `@SuppressWarnings("SameParameterValue")`.

Reply:
going to loop through supportedVersions()